.github/workflows/release.yml

```diff
@@ -43,3 +42,5 @@ jobs:
           SIGNORE_CLIENT_ID: ${{ secrets.SIGNORE_CLIENT_ID }}
           SIGNORE_CLIENT_SECRET: ${{ secrets.SIGNORE_CLIENT_SECRET }}
           SIGNORE_SIGNER: ${{ secrets.SIGNORE_SIGNER }}
+permissions:
+  contents: read
```

Comment:
Is it just read? Go releaser will need permission to publish the actual release pkg. 

Also for permissions we generally add the block towards the top of the file before the jobs section to make it easier to see what permissions are needed by the action.